HoloPipelines/pipelines/components/compNumpyTransformation.py

```diff
@@ -0,0 +1,47 @@
+import scipy.ndimage
+
+
+def crop(img, cropx, cropy, cropz):
+    x, y, z = img.shape
+
+    startX = x // 2 - (cropx // 2)
+    startY = y // 2 - (cropy // 2)
+    startZ = z // 2 - (cropz // 2)
+
+    endX = startX + cropx
+    endY = startY + cropy
+    endZ = startZ + cropz
+
+    return img[startX:endX, startY:endY, startZ:endZ]
+
+
+def resize(img, ratioX, ratioY, ratioZ):
+    return scipy.ndimage.interpolation.zoom(img, [ratioX, ratioY, ratioZ])
```

Comment:
If this one line function is used only once in this file. It can be called solely on its own.

Reply:
removed resize()